docs/usage/CATEGORY_SYNC.md

```diff
@@ -138,4 +142,26 @@ More examples of those utils for different fields can be found [here](/src/integ
 ## Caveats
 
 1. Categories are either created or updated. Currently the tool does not support category deletion.
-2. The library will sync all field types of custom fields, except `ReferenceType`. [#87](https://github.com/commercetools/commercetools-sync-java/issues/3). 
+2. The sync library is not meant to be executed in a parallel fashion. For example:
+    ````java
```

Comment:
...parallel way

Reply:
I think "parallel way" sound better.

Reply:
Hmm. I am also more leaning towards "fashion" 😬 Anyways, I think it's not very affecting the meaning. So I will leave it like this for now. Ok? @ahalberkamp